### DISCUSSION TITLE
RuntimeError NVIDIA driver too old

### DISCUSSION THREAD

**Discussion by gpk3** [Acly/krita-ai-diffusion#1105] [Q&A] [answered]
When I launch AI Image Generation using my GPU, I get this error.

RuntimeError: The NVIDIA driver on your system is too old (found version 11040). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver.
2024-08-23 20:26:50,282 ERROR 

![Error Msg](https://github.com/user-attachments/assets/72d39411-0946-4cb6-85ba-574a49b3d337)

![nvidia smi](https://github.com/user-attachments/assets/a8e6ff82-905b-4893-a81f-b95b72bfb79f)

My GPU is a GTX 680 and my drivers are current. Is there anything I can do to launch AI Image Generation using my GPU or is my graphics card simply too old? If I do need to upgrade my graphics card, which one would you recommend? Thank you

**Comment by Acly** [comment, marked as answer]
Anything below GTX 9xx generation is not supported, and I think some features require 10xx.

> If I do need to upgrade my graphics card, which one would you recommend?

RTX 4060 Ti 16GB seems good value, but all the RTX 30xx and 40xx will work great with current models, and anything else is speculation.

**Reply by gpk3** [reply]
Thank you!
